### PR TITLE
Drawing/add modal

### PR DESCRIPTION
**Pull request by noahmalmed**
Invision Mock-ups: https://projects.invisionapp.com/d/main#/console/13393648/320911080/preview

# Description:
This pull request adds the drawing modal. This is all just UI stuff and Navigation. The next PR will contain the actual drawing.

A couple of things to note: 
 - Some of the components added are from the prototype
 - All navigation on the modal should send you back to the classifier screen

I would look at this PR by the commits, most have some description messages.

# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/common/CloseButton.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            onPress={props.onPress}
+            style={[props.style, {height: props.size, width: props.size}]}
+            activeOpacity={0.3}
+        >
+            <View
+                style={[backgroundViewSize, styles.circle]}
+            />
+            <Icon
+                style={styles.icon}
+                name="times-circle"
```
Comment (wgranger) [inline src/components/common/CloseButton.js]: Is this name necessary to find the correct shape?

**Comment by wgranger** [comment]
screen shot 2018-10-09 at 2 48 30 pm

I'm getting this error. Can you confirm that you're still able to do a fresh build?

File: src/components/Markings/components/MarkableImage.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+// import { bindActionCreators } from 'redux'
+// import * as drawingScreenAction from '../../../actions/drawingScreen'
+
+// const mapStateToProps = (state) => ({
+    // shapes: state.drawingScreen.shapes
+// })
+
+// const mapDispatchToProps = (dispatch) => ({
+//      drawingScreenActions: bindActionCreators(drawingScreenAction, dispatch)
+// })
```
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/components/Markings/components/MarkableImage.js]: These are going to all be commented out in the next PR. I didn't want to get to ahead of myself with what is being committed

**Comment by noahmalmed** [comment]
@beckyrother Any thoughts on Will's comment

**Comment by beckyrother** [comment]
It does feel a little awkward to have to click twice; on the other hand, it's nice to click through in order to have as large of a working area as possible.

What if we make it very clear that tapping the image the first time will enlarge it. Part of the frustration here is that it looks like you're able to make annotations immediately.

https://invis.io/CTO5DBHSJ2D#/320911083_Projects_-_Nature_-_Drawing_1

Add a little blur and the font-awesome `fa-expand-arrows-alt` icon with a little drop shadow on the image so it feels like you have to click it before you can take any other actions.

**Comment by noahmalmed** [comment]
And once there are drawings, should we show those on the non-modal screen instead of the expand/blur view

**Comment by beckyrother** [comment]
Would it be possible to still show them but they be blurred as well?

**Comment by noahmalmed** [comment]
👍 Thanks Becky. @wgranger I'm going to implement these changes in a different PR bc it will be a decent chunk of work

**Comment by wgranger** [comment]
Sounds good. I'm fine with the rest of this code as is.

**Comment by beckyrother** [comment]
Thanks for pointing this out! Hooray teamwork

**Pull request merged**